.github/workflows/mcp_unit_test.yml

```diff
@@ -0,0 +1,65 @@
+name: 'MCP Unit Tests'
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  terraform:
+    name: 'MCP Unit Tests'
+    runs-on: ubuntu-latest
+    if: contains(github.head_ref, 'MCP') == true
```

Comment:
Make this, startsWith 'feat/', 'bug/', 'patch/'

Reply:
Changed this to:
`contains(github.head_ref, 'MCP') &&
(startswith(github.head_ref, 'feat') ||
startswith(github.head_ref, 'bug') ||
startswith(github.head_ref, 'patch'))` 

As it still needs to only run for MCP branches

Reply:
As mentioned earlier, don't make this just about mcp. Probably should rename the file as well